examples/explorer/README.md

```diff
@@ -0,0 +1,29 @@
+# Basic Explorer
+
+A simple Cardano Explorer using Reactjs, React-bootstrap and Relay to use with Jorgunmandr node.
+
+# Prerequesites
+
+- You need a Jormungandr node running in explorer mode in order to use this app.
```

Comment:
link to the documentation here: https://input-output-hk.github.io/jormungandr/quickstart/04_explorer.html